odd-platform-ui/src/redux/selectors/datasetStructure.selectors.ts

```diff
@@ -64,42 +64,27 @@ export const getDatasetStructureTypeStats = createSelector(
   }
 );
 
-const getDatasetFieldLabelsUpdatingStatus = createFetchingSelector(
-  'PUT_DATA_SET_FIELD_LABELS'
-);
-
-export const getDatasetFieldLabelsUpdating = createSelector(
-  getDatasetFieldLabelsUpdatingStatus,
-  status => status === 'fetching'
-);
-
 export const datasetFieldId = (_: RootState, fieldId: number) => fieldId;
 
-export const getDatasetFieldLabels = createSelector(
-  getDatasetStructureState,
-  datasetFieldId,
-  (datasetStructureState, fieldId) => {
-    if (!fieldId) return [];
-    return datasetStructureState.fieldById[fieldId]?.labels || [];
-  }
-);
-
-const getDatasetFieldDescriptionUpdatingStatus = createFetchingSelector(
-  'PUT_DATA_SET_FIELD_INTERNAL_DESCRIPTION'
+const getDatasetFieldFormDataUpdatingStatus = createFetchingSelector(
+  'PUT_DATA_SET_FIELD_FORM_DATA'
 );
 
-export const getDatasetFieldDescriptionUpdating = createSelector(
-  getDatasetFieldDescriptionUpdatingStatus,
+export const getDatasetFieldFormDataUpdating = createSelector(
+  getDatasetFieldFormDataUpdatingStatus,
   status => status === 'fetching'
 );
 
-export const getDatasetFieldInternalDescription = createSelector(
+export const getDatasetFieldFormData = createSelector(
```

Comment:
Looks like this is current state of DatasetField rather than form data.

Reply:
Yes, renamed it